www/js/control/ProfileSettings.jsx

```diff
@@ -187,13 +201,13 @@ const ProfileSettings = () => {
     const newNotificationSettings = {};
 
     if (uiConfig?.reminderSchemes) {
-      const prefs = await NotificationScheduler.getReminderPrefs();
+      const prefs = await getReminderPrefs();
```

Comment:
This function takes reminderSchemes as a parameter, doesn't it?

```suggestion
      const prefs = await getReminderPrefs(uiConfig.reminderSchemes);
```

Reply:
I did forget this, good catch! It was working in my testing without this, because `reminderSchemes` only gets used in `getReminderPrefs()` if the user object does not already exist on the server and it needs to do the initialization of notification-scheduling for a new user